src/store/index.js

```diff
@@ -78,6 +91,18 @@ export default new Vuex.Store({
           commit('setPreferencesError', error);
         });
     },
+    fetchOrganizations({ commit }) {
+      commit('setFetchingOrganizations', true);
+      organizationsApi.getOrganizations()
+        .then(response => {
+          commit('setFetchingOrganizations', false);
+          commit('setOrganizations', response.data.data);
+        })
+        .catch(error => {
+          commit('setFetchingOrganizations', false);
+          commit('setOrganizationsError', error);
+        });
```

Comment:
Muy menor y opcional, pero puedes usar [`finally`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally) acá:

```suggestion
      organizationsApi.getOrganizations()
        .then(response => {
          commit('setOrganizations', response.data.data);
        })
        .catch(error => {
          commit('setOrganizationsError', error);
        })
        .finally(() => {
          commit('setFetchingOrganizations', false);
        });
```

Reply:
esto lo dejaré para un pr de mejoras chicas, para juntar todo